lib/segment/src/index/field_index/btree_index.rs

```diff
@@ -46,6 +66,25 @@ impl<T: KeyEncoder> NumericIndex<T> {
             self.add_value(idx, value);
         }
     }
+
+    #[allow(dead_code)] // TODO(gvelo): remove when this index is integrated in `StructPayloadIndex`
+    fn load(&mut self) {
+        let db_ref = self.db.borrow();
+        let cf_handle = db_ref.cf_handle(&self.store_cf_name).unwrap();
+        let mut iter = db_ref.raw_iterator_cf(&cf_handle);
```

Comment:
I used `raw_iterator_cf` because according to the rocksdb documentation they tend to have better  performance compared to `iterator_cf` , the downside of `raw_iterator_cf` is that it depart from the rust iterator idiom. If keeping the rust idiom is a concern here  let's use the regular db iterator.

Reply:
I would keep as is, there no much changes to call it premature optimization